sci-biology/bx-python/bx-python-0.13.0.ebuild

```diff
@@ -17,6 +17,7 @@ LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
+BDEPEND="test? ( dev-python/python-lzo )"
```

Comment:
```suggestion
BDEPEND="test? ( dev-python/python-lzo[${PYTHON_USEDEP}] )"
```

Reply:
Uh, sorry - shouldn't have missed that! Thanks